src/api/entities/Participant.ts

```diff
@@ -59,7 +59,7 @@ export class Participant extends BaseModel {
 
 // TODO: Can ModelObjectOpt do relationships automatically?
 export type ParticipantDTO = Omit<ModelObjectOpt<Participant>, 'types' | 'users'> & {
-  types?: ModelObjectOpt<ParticipantType>[];
+  types?: ParticipantTypeDTO[];
```

Comment:
what's the rationale behind this change?

Reply:
Just because we're already doing the ModelObjectOpt<> to create ParticipantTypeDTO anyway - I think the types would have just "matched", but I'm used to C# where types which are just the same still won't match unless they're the exact same type.